library/core/src/intrinsics.rs

```diff
@@ -2717,7 +2717,7 @@ pub const unsafe fn copy<T>(src: *const T, dst: *mut T, count: usize) {
     // SAFETY: the safety contract for `copy` must be upheld by the caller.
     unsafe {
         assert_unsafe_precondition!(
-            "ptr::copy requires that both pointer arguments are aligned aligned and non-null",
+            "ptr::copy requires that both pointer arguments are aligned and non-null",
```

Comment:
sometimes duplicated words are intentional. I don't think this is one such case, although iirc someone warned me that `mutable mutable` (or was it `immutable immutable`?) might be a thing... (and there's one case where `optional optional` is sort of a thing).

Reply:
This one is a typo. Good find.